### PR TITLE
chore: add issue and PR templates

### PR DESCRIPTION
**Pull request by kenji-miyake**
Added templates.
Please refer to https://github.com/kenji-miyake/test-pr-template about its behavior.

References:
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
- https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

### CODE REVIEW

**Review by isamu-takagi — DISMISSED** [review]
Great, I confirmed the behavior in the demo repository.

**Comment by kenji-miyake** [comment]
Thank you!
I'll wait for @xmfcx or @mitsudome-r to approve this.

File: .github/PULL_REQUEST_TEMPLATE/small-change.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Reviewers **must** check the checkboxes below before approval.
+
+- [ ] The PR follows the [pull request guidelines].
+
+## Post-review checklist for the PR author
+
+PR author **must** check the checkboxes below before merging.
+
+- [ ] There are no open points or they are tracked via tickets.
```
Comment (xmfcx) [inline .github/PULL_REQUEST_TEMPLATE/small-change.md]:
```suggestion
- [ ] There are no open discussions or they are tracked via tickets.
```

**Review by xmfcx — CHANGES_REQUESTED** [review]
I've also tested it in a fork and seems to be working well. I hope people can read and actually press the `Preview` button and select a template.

That feels so hacky, surprised github doesn't provide native support for this.

File: .github/PULL_REQUEST_TEMPLATE/standard-change.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+- [ ] The PR follows the [pull request guidelines].
+- [ ] The PR has been properly tested.
+- [ ] The PR has been reviewed by the code owners.
+
+## Post-review checklist for the PR author
+
+PR author **must** check the checkboxes below before merging.
+
+- [ ] There are no open points or they are tracked via tickets.
```
Comment (xmfcx) [inline .github/PULL_REQUEST_TEMPLATE/standard-change.md]:
```suggestion
- [ ] There are no open discussions or they are tracked via tickets.
```

**Review by xmfcx — CHANGES_REQUESTED** [review]
Missed this one.

**Comment by kenji-miyake** [comment]
FYI, I'll link a thread on GitHub Community Forum.
https://github.community/t/is-there-a-pull-request-template-selector-similar-to-issues/171838

**Review by xmfcx — APPROVED** [review]

**Pull request merged**